docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,191 @@
+|`type` |String a|Must be `simple`. |Yes
+|`_version` |String a|The version id, normally returned by the API when the item was retrieved. Use it ensure updates are done against the latest version. |No
+
+|==============================================
+// end::update-exception-item-request-body[]
+
+
+
+// tag::api-id-or-itemid-query-params[]
+The URL query must include one of the following:
+
+* `id` - the `id` of the item, or
+* `item_id` -  the `item_id` of the item
+
+In addition to the above, `namespace_type` URL query parameter is also required with a value of `agnostic`
+// end::api-id-or-itemid-query-params[]
+
+
+// tag::find-exception-items-query-params[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
+|`namespace_type` |String a|Must be set to `agnostic`. |Yes
+|`page` |Integer |The page number to return. |No
+|`per_page` |Integer |The number of items to return per page. |No
+|`sort_field` |String |Determines which field is used to sort the results. |No
+|`sort_order` |String a|Determines the sort order, which can be `desc` or `asc`. |No
+|`filter` |String |A {kibana-ref}/kuery-query.html[{kib} Query Language (KQL)] string to filter the results down. |No
+
+|==============================================
+// end::find-exception-items-query-params[]
+
+
+
+// tag::endpoint-exceptions-value-types-info-section[]
+==== Value types
+
+The following describes the types that can be defined when using the xref:create-item-api[create] or xref:update-item-api[update] {endpoint-artifact-name} APIs:
+
+[#exception-item-comment-schema]
+===== `comment` object schema
+
+Comments are JSON objects containing the following structure:
+
+[source,json]
+---------
+{
+    "comment": "some comment here"
+}
+---------
+
+When used with the update API, existing comments can be updated by using their associated `id`, while any comment without the `id` attribute will be added as a new comment:
+
+[source,json]
+---------
+{
+    "comment": "some comment here - updated",
+    "id": "1078cf59-5893-4143-acf7-40a40af16bee"
+}
+---------
+
+[#exception-item-os-types]
+===== `os_types` values
+
+The following are the valid OS types that can be used when creating {endpoint-artifact-name}:
+
+* `windows`
+* `linux`
+* `macos`
+
+[#exception-item-scope-assignment-tags]
+===== Scope assignment
+
+{endpoint-artifact-name-sentence-start} can be assigned globally across all endpoint policies, or assigned to specific policies. The assignment of a {endpoint-artifact-name-single} is done by defining one or more tags with a prefix of `policy:`. Note that the {endpoint-artifact-name-single} can be either "global" or "per policy", but not *both*. The following tags are available for use in order to control the assignment scope:
```

Comment:
Revise to avoid conflict with "a/an" in other artifact name (like "event filter").
```suggestion
{endpoint-artifact-name-sentence-start} can be assigned globally across all endpoint policies, or assigned to specific policies. The assignment of the {endpoint-artifact-name-single} is done by defining one or more tags with a prefix of `policy:`. Note that the {endpoint-artifact-name-single} can be either "global" or "per policy", but not *both*. The following tags are available for use in order to control the assignment scope:
```